include/interface/platform/File.h

```diff
@@ -34,6 +34,12 @@ struct File {
 
 	virtual auto size() -> size_t = 0;
 
+	virtual auto no() -> size_t = 0;
+
+	virtual auto path(uint32_t fd, char *bufferpath, uint32_t size) -> size_t = 0;
+
+	virtual auto path(uint32_t fd, std::span<char> bufferpath) -> size_t = 0;
```

Comment:
même question que sur l'autre PR, est-ce que la version `std::span` ne suffit pas?

```suggestion
	virtual auto path(uint32_t fd, const char *path, uint32_t size) -> size_t = 0;

	virtual auto path(uint32_t fd, std::span<const char> path) -> size_t = 0;
```